.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,44 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ['main']
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: accenture/federated-catalogue-viewer
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
```

Comment:
Do we need all those?

Reply:
[243efb0](https://github.com/Accenture/Federated-Catalogue-Viewer/pull/2/commits/243efb0d16c3c6d9f4ea8f3e622462699bb43f7e)